cli/azd/resources/scaffold/templates/host-containerapp.bicept

```diff
@@ -51,7 +51,7 @@ resource containerRegistry 'Microsoft.ContainerRegistry/registries@2022-02-01-pr
   name: containerRegistryName
 }
 
-resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2023-04-01-preview' existing = {
+resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2023-04-01' existing = {
```

Comment:
Should this one be `2023-05-01` as well?

Reply:
I need to investigate about this